PostmanCollections/Payment Hub.json

```diff
@@ -43307,7 +43405,7 @@
 							}
 						],
 						"url": {
-							"raw": "{{OperationsHostName}}/api/v1/batch?batchId={{batchId}}",
+							"raw": "{{OperationsHostName}}/api/v1/batch?batchId={{batchId}}&X-CorrelationID={{clientCorrelationId}}",
```

Comment:
why are you passing correlation id in the URL?

Reply:
I just added in the header, this gets added to the url automatically

Reply:
that is strange